src/plugins/home/public/plugin.ts

```diff
@@ -42,10 +42,12 @@ export interface HomePluginStartDependencies {
   dataViews: DataViewsPublicPluginStart;
   urlForwarding: UrlForwardingStart;
   guidedOnboarding?: GuidedOnboardingPluginStart;
+  cloudStart: CloudStart;
+  shareStart: SharePluginStart;
```

Comment:
I _think_ these have to match the plugin's internally-declared ID:

```suggestion
  cloud: CloudStart;
  share: SharePluginStart;
```

Reply:
Cloud is already used in getting_start.tsx T_T